src/components/common/Modal.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="fixed z-50 top-0 left-0 bg-black bg-opacity-60 h-screen w-screen">
-    <div class="flex justify-center items-center w-full h-full">
+    <div
+      @click.self="$emit('closeModal')"
+      class="flex justify-center items-center w-full h-full"
+    >
       <div class="relative flex flex-col bg-white">
         <button @click="$emit('closeModal')" class="w-4 absolute top-3 right-4">
```

Comment:
@click.stop żeby zatrzymać propagacje na klikaniu, ale możlwe, że tak też działa

Reply:
`@click.stop` nie działa, event `closeModal` jest rejestrowany również jak klikniesz input, a nie tylko jak klikniesz obszar poza modalem

Reply:
A dobra teraz ogarnąłem czym to jest, myślałem, że próbujesz zatrzymać dwa emity, na kliknięciu wewnętrznego buttona :P